identity/client/src/pages/authorizations/modals/AddModal.tsx

```diff
@@ -177,14 +174,16 @@ const AddModal: FC<UseEntityModalProps<ResourceType>> = ({
       <CheckboxGroup
         legendText={
           <PermissionsSectionLabel>
-            <Translate>Select at least one permission. Visit</Translate>{" "}
-            <DocumentationLink
-              path="/concepts/resource-authorizations/"
-              withIcon
-            >
-              {t("Resource permissions")}
-            </DocumentationLink>{" "}
-            <Translate>for a full overview.</Translate>
+            <Translate i18nKey="selectPermission">
```

Comment:
`Translate` is only used when we have child elements. in this case `DocumentationLink`

everywhere else, we need to stick to `t('translationKey')`